docs/crypto.rst

```diff
@@ -11,18 +11,22 @@ Simple TEAL opcodes have cost `1`, and more advanced cryptographic operations ha
 cost. Below is how you express cryptographic primitives in PyTeal:
```

Comment:
@algoidurovic Minor:  The paragraph that's out of view from the PR presents stale info because it references how many elements appear in the table.

Open to customization - I suggest simplifying to something like:

> Algorand Smart Contracts support these cryptographic primitives.  The table below discusses usage and presents opcode cost.

Reply:
@algoidurovic Thanks for the change in https://github.com/algorand/pyteal/pull/307/commits/59d811948250975c7e26f6d077900de2c500900e.  

I think in a future iteration we can make a broader pass through `crypto.rst`.  For the PR, it looks reasonable to me - resolving.